### DISCUSSION TITLE
can anyone make a playbook without internet?thx

### DISCUSSION THREAD

**Discussion by jeojeo1225** [meetrevision/playbook#134] [Q&A]
becuse china has GFW,i cant connect github and some web without vpn.
i use the playbook then i get lot of error and timeout

**Comment by jeojeo1225** [comment]
oh ,i find it out. config the playbook.conf 
    <Requirements>
        <Requirement>DefenderToggled</Requirement>
        <Requirement>NoPendingUpdates</Requirement>
        <Requirement>NoAntivirus</Requirement>
        
        <Requirement>PluggedIn</Requirement>

**Comment by jeojeo1225** [comment]
<Requirements>
        <Requirement>DefenderToggled</Requirement>
        <Requirement>NoPendingUpdates</Requirement>
        <Requirement>NoAntivirus</Requirement>
        
        <Requirement>PluggedIn</Requirement>

**Comment by Pyenb** [comment]
Yes, that will work. But also remember that the Playbook will still try to download the Revision tool, so it could still throw some errors.

**Reply by jeojeo1225** [reply]
Thx for you notice.
